restler/engine/transport_layer/messaging.py

```diff
@@ -77,7 +80,7 @@ def _get_method_from_message(self, message):
         method_name = message[0:end_of_method_idx]
         return method_name
 
-    def sendRecv(self, message, req_timeout_sec):
+    def sendRecv(self, message, req_timeout_sec=600):
```

Comment:
>600 [](http://example.com/codeflow?start=48&length=3)

please sync to the latest in main and rebase - this should not be part of the changes